apps/etheroscope_db/config/test.exs

```diff
@@ -0,0 +1,5 @@
+use Mix.Config
+
+# Configure your database
+config :etheroscope_db,
+  database: "etheroscope_test"
```

Comment:
Could you add a staging db too?

Reply:
You can set staging through a system variable now (`COUCHDB_ENV`).